dist/_include-media.scss

```diff
@@ -237,6 +237,7 @@ $im-no-media-expressions: ('screen', 'portrait', 'landscape') !default;
       $value: get-expression-value($condition, $operator);
 
       // scss-lint:disable SpaceAroundOperator
+      // Long if statement
```

Comment:
I’d suggest we just disable that rule entirely. No point in having it if we keep fighting against it.